src/screens/proposals/components/toggle/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Switch } from '@material-ui/core';
+import { useGetScreenSizeHook } from '@hooks';
+import classnames from 'classnames';
+import { useGetStyles } from './styles';
+import { ToggleProp } from './types';
+
+const Toggle = (props: ToggleProp) => {
```

Comment:
why is there props but all i see it `<Toggle />` If you can explain this to me then you don't have to fix